internal/host/validator.go

```diff
@@ -819,6 +820,20 @@ func (v *validator) isNTPSynced(c *validationContext) (ValidationStatus, string)
 	return status, message
 }
 
+// TODO think about name
+func (v *validator) isTimeSyncedBetweenHostAndService(c *validationContext) (ValidationStatus, string) {
+	if c.host.Timestamp == 0 {
+		return ValidationPending, "Missing timestamp"
+	}
+
+	diff := time.Now().UTC().Sub(time.Unix(c.host.Timestamp, 0).UTC())
```

Comment:
If the host time is 1 hour before the service time it means that for about an hour bootkube will fail for about an hour which will most likely fail the installation (some timeout will pop).
I don't think we want that.
We can allow the host clock to be a few minutes behind (~10) and an hour ahead.